src/content/app/genome-browser/components/browser-sidebar-toolstrip/BrowserSidebarToolstrip.test.tsx

```diff
@@ -83,93 +86,101 @@ describe('<TrackPanelBar />', () => {
         set(
-          `browser.trackPanel.${fakeGenomeId}.trackPanelModalView`,
-          '',
+          `browser.browserSidebarModal.${fakeGenomeId}.browserSidebarModalView`,
+          null,
           mockState
         )
       );
       const bookmarksButton = [...container.querySelectorAll('button')].find(
-        (button) => button.innerHTML === TrackPanelModalView.BOOKMARKS
+        (button) => button.innerHTML === BrowserSidebarModalView.BOOKMARKS
       ) as HTMLButtonElement;
 
       userEvent.click(bookmarksButton);
 
-      const toggleTrackPanelAction = store
+      const toggleBrowserSidebarModalAction = store
         .getActions()
         .find(
           (action) =>
-            action.type === trackPanelActions.updateTrackPanelForGenome.type
+            action.type ===
+            browserSidebarModalActions.updateBrowserSidebarModalForGenome.type
         );
 
       const expectedPayload = {
         activeGenomeId: fakeGenomeId,
         data: {
-          ...mockState.browser.trackPanel[fakeGenomeId],
-          trackPanelModalView: TrackPanelModalView.BOOKMARKS,
-          isTrackPanelModalOpened: true
+          ...mockState.browser.browserSidebarModal[fakeGenomeId],
+          browserSidebarModalView: BrowserSidebarModalView.BOOKMARKS,
+          isBrowserSidebarModalOpened: true
         }
       };
 
-      expect(toggleTrackPanelAction.payload).toEqual(expectedPayload);
+      expect(toggleBrowserSidebarModalAction.payload).toEqual(expectedPayload);
     });
 
     it('opens the track panel if it is closed when a button is clicked', () => {
-      const { container } = renderComponent(
-        set(
-          `browser.trackPanel.${fakeGenomeId}.isTrackPanelOpened`,
-          false,
-          mockState
-        )
-      );
+      const newMockState = merge(mockState, {
+        browser: {
+          browserSidebarModal: {
+            [fakeGenomeId]: {
+              isBrowserSidebarModalOpened: false,
+              browserSidebarModalView: null
+            }
+          }
+        }
+      });
+
+      const { container } = renderComponent(newMockState);
       const bookmarksButton = [...container.querySelectorAll('button')].find(
-        (button) => button.innerHTML === 'Previously viewed'
+        (button) => button.innerHTML === BrowserSidebarModalView.BOOKMARKS
       ) as HTMLButtonElement;
 
       userEvent.click(bookmarksButton);
-      const toggleTrackPanelAction = store
+
+      const toggleBrowserSidebarModalAction = store
         .getActions()
         .find(
           (action) =>
-            action.type === trackPanelActions.updateTrackPanelForGenome.type
+            action.type ===
+            browserSidebarModalActions.updateBrowserSidebarModalForGenome.type
         );
 
       const expectedPayload = {
         activeGenomeId: fakeGenomeId,
         data: {
-          ...mockState.browser.trackPanel[fakeGenomeId],
-          isTrackPanelOpened: true
```

Comment:
This test was also checking that the track panel itself gets open:

https://user-images.githubusercontent.com/6834224/163131612-9ecfee97-cd8c-40f9-843d-2471483b251f.mp4

Reply:
✅